### PR TITLE
Added extra steps available and some refactoring

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
@PVince81 The steps you requested.

### CODE REVIEW

File: tests/integration/features/bootstrap/Provisioning.php [hunk 1]
```diff
@@ -24,40 +24,59 @@
...
+	/**
+	 * @Then /^user "([^"]*)" already exists$/
+	 * @param string $user
+	 */
+	public function userAlreadyExists($user) {
+		PHPUnit_Framework_Assert::assertTrue($this->userExists($user));
+	}
+
+	/**
+	 * @Then /^user "([^"]*)" does not already exist$/
```
Comment (PVince81) [inline tests/integration/features/bootstrap/Provisioning.php]: @phil-davis strange wording ? better alternative ?

**Review by phil-davis — APPROVED** [review]
LGTM

**Comment by PVince81** [comment]
@SergioBertolinSG thanks

File: tests/integration/features/bootstrap/Provisioning.php (hunk 1 above)
Comment (phil-davis) [reply tests/integration/features/bootstrap/Provisioning.php]: It sounds a bit strange, but I can't think how else to say it better.
We need to keep "Given user XYZ does not exist" (1) to be the opposite of "Given user XYZ exists" (2) - and (2) creates the user if it is not there, so (1) should delete the user if it exists.

**Comment by PVince81** [comment]
@SergioBertolinSG please backport to stable10

**Pull request merged**